gaurun/worker.go

```diff
@@ -3,53 +3,94 @@ package gaurun
 import (
 	"github.com/RobotsAndPencils/buford/push"
 	"strings"
+	"sync/atomic"
 )
 
+var (
+	PusherCount int64
```

Comment:
IIUC, pushNotificationWorker has own `PushCount` which limits the number of goroutine to send push in parallel at the same time in the worker.
This `PushCount` is shared by all pushNotificationWorker, so it limits the all workers' goroutine.